src/components/equipment/BarcodeScanner.tsx

```
@@ -0,0 +1,178 @@
+// Admin barcode scanner for checking equipment in and out.
+// Uses the device camera via the browser's BarcodeDetector API.
+
+import { useState } from "react";
+import { ScanLine, CheckCircle2, XCircle, AlertTriangle, Camera } from "lucide-react";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import {
+    checkOutEquipment,
+    checkInEquipment,
+} from "@/lib/equipment";
```

ESLint

[error] Replace `⏎····checkOutEquipment,⏎····checkInEquipment,⏎` with `·checkOutEquipment,·checkInEquipment·`

(prettier/prettier)